docker-compose.yml

```diff
@@ -3,9 +3,12 @@ version: '2.1'
 services:
   file-service:
     image: file-service
-    build: .
+    build: 
+      context: .
+      dockerfile: test.Dockerfile
     environment:
       NODE_ENV: prod
```

Comment:
NODE_ENV should be set to `production` instead, according to: [Performance Best Practices Using Express in Production](https://expressjs.com/en/advanced/best-practice-performance.html#set-node_env-to-production)

Reply:
@AllKinds